lutron_caseta_pro/__init__.py

```diff
@@ -46,6 +47,7 @@
             {
                 vol.Required(CONF_HOST): string,
                 vol.Optional(CONF_MAC): string,
+                vol.Optional(CONF_TRANSITION_TIME): positive_int,
```

Comment:
Can you make this float? Transition will support fractional amounts like 0.5 seconds.

Lutron has a max transition time of 4 hours or 14400 s, but looking at the light platform, it uses [this](https://github.com/home-assistant/home-assistant/blob/7807b409258864a2aa35e25f2f6aacc416429319/homeassistant/components/light/__init__.py#L89) so it may be safer:
```python
vol.All(vol.Coerce(float), vol.Clamp(min=0, max=6553))
```
Alternatively, could also import `VALID_TRANSITION` from light and use that directly.

Reply:
Good call - not being familiar with the HA codebase, I didn't know about VALID_TRANSITION. Using that is easier and cleaner to read.